falcon/media/multipart.py

```diff
@@ -578,6 +581,13 @@ class MultipartParseOptions:
     :class:`.MultipartParseError` will be raised. If this option is set to 0,
     no limit will be imposed by the parser.
     """
+    max_secure_filename_length: Optional[int]
+    """The maximum number characters for a secure filename (default ``96``).
+
+    If the form contains more parts than this number, an instance of
```

Comment:
this doc seems not correct. It should probably mention that the name will simply be truncated

Reply:
Aye, this looks like copypaste from another field's docstring.

Reply:
Addressed.